### DISCUSSION TITLE
OCR選項為什麼沒看到tesseract？

### DISCUSSION THREAD

**Discussion by lp43** [pot-app/pot-desktop#326] [Q&A] [answered]
請問OCR選項為什麼沒看到tesseract？
1.11.0Windows １０版本

**Comment by Pylogmon** [comment]
1.11版本开始删除了Tesseract，用系统OCR代替。

**Reply by lp43** [reply]
請問系統OCR是什麼，我使用它沒反應。要安裝什麼嗎

**Reply by Pylogmon** [reply, marked as answer]
Windows系统提供的OCR，默认只有系统语言的支持，要支持其他语言可以单独安装语言包，麻烦把软件升级到最新版本，不支持的语言会有相关说明。

**Reply by lp43** [reply]
Tesseract(offline)功能可以放回來嗎？

1. 許多內地的OCR KEY的取得都要實名驗證
2. 實名驗證的過程需要綁大陸手機號或台胞證，這不是每個人都有的東西
3. 上述流程讓Pot Translate使用到OCR服務變得相對困難。

舉例，百度OCR要實名驗證，在輸入手機號時鬼打牆永遠綁定不成功，我在猜應該是因為台灣手機號不被允許，所以根本申請不了百度OCR。

**Reply by Pylogmon** [reply]
Windows的系统OCR无论是速度还是质量都比Tesseract要好，目前Linux下的系统OCR是通过Tesseract来实现，但是更改了实现方式，这一方式Windows无法使用。

**Reply by Pylogmon** [reply]
升级最新版后如果你的系统OCR还是无法使用麻烦提交issue反馈报错，后续的离线OCR主要会借助系统的OCR来完善

**Reply by lp43** [reply]
已提交issue。 [單號#328](https://github.com/pot-app/pot-desktop/issues/328)
